examples/trivial-okapi/TenantList.js

```diff
@@ -28,8 +28,14 @@ class TenantList extends Component {
     if (!data.tenants) return null;
     const tenantNodes = data.tenants.map(tenant => (
       <li key={tenant.id}>
-        {tenant.name} [<button tabIndex={0} onClick={() => mutator.tenants.DELETE(tenant)}>delete</button>]
-          [<Link to={`${pathname}/edit/${tenant.id}`}>Edit</Link>]
+        {tenant.name}
+        {' '}
+        [
+        <button type="button" tabIndex={0} onClick={() => mutator.tenants.DELETE(tenant)}>delete</button>
+        ]
+        [
+        <Link to={`${pathname}/edit/${tenant.id}`}>Edit</Link>
+        ]
```

Comment:
Is there a reason for this array notation here?

Reply:
No. Removed brackets.